config/samples/example_flags.yaml

```diff
@@ -0,0 +1,88 @@
+flags:
```

Comment:
I converted example_flags.json to yaml using https://www.json2yaml.com/ and tweaked the yaml a bit.